.claude/skills/ship-release/SKILL.md

```
@@ -94,29 +107,107 @@ All commands in this phase run inside `$RELEASE_WT` (`git -C "$RELEASE_WT" ...`
+
+   ```sh
+   curl -s -u "$LOGIN:$TOKEN" "https://rocketchat.atlassian.net/rest/api/3/project/CORE/version?maxResults=100&query=Electron"
+   ```
+
+   Releases are usually pre-created by the team. If absent, ask — creating one needs project-admin rights (see step 6).
+
+2. **Inventory.** For each PR in the Phase 1 list, find its Jira issue: a `CORE-NNNN` in the PR title/body/branch, or search by feature keywords (`searchJiraIssuesUsingJql`, component `Electron`). Broad JQL sweeps blow the MCP token limit — restrict `fields` to `summary,status,issuetype,fixVersions` and parse the saved file with `ctx_execute` when it still overflows.
+
+3. **Existing issues** — add the fixVersion, leaving other fields alone:
+
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)